frontend/src/wizards/Governance/Policy/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml

```diff
@@ -17,79 +17,25 @@ spec:
   disabled: false
   policy-templates:
     - objectDefinition:
-        apiVersion: policy.open-cluster-management.io/v1
-        kind: ConfigurationPolicy
+        apiVersion: policy.open-cluster-management.io/v1beta1
+        kind: OperatorPolicy
         metadata:
-          name: comp-operator-ns
+          name: operatorpolicy-comp-operator
         spec:
-          remediationAction: inform # will be overridden by remediationAction in parent policy
+          remediationAction: inform
           severity: high
-          object-templates:
-            - complianceType: musthave
-              objectDefinition:
-                apiVersion: v1
-                kind: Namespace
-                metadata:
-                  name: openshift-compliance
```

Comment:
Shouldn't the policy still create this namespace?

Reply:
I was under the impression that the current version of the controller will create the ns if it doesn't already exist in enforce mode. Is this not the case?

Reply:
It doesn't.